### PR TITLE
Pranita_Assignment_37_Done_30mins

### PR DESCRIPTION
**Pull request by PranitaPJun20**
image

### CODE REVIEW

File: src/pranitaPJun20/MaxFreqWordString_Collections.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+public class MaxFreqWordString_Collections {
+
+	void findMaxRepeatedWordFrequency(String input) {
+		String[] inputArray = input.split(" ");
+		int count = 0;
+		Map<String, Integer> wordMap = new LinkedHashMap<String, Integer>();
+		for (String name : inputArray) {
+			if (wordMap.containsKey(name)) {
+				wordMap.put(name, wordMap.get(name) + 1);
```
Comment (harshvegada) [inline src/pranitaPJun20/MaxFreqWordString_Collections.java:28]: Hi @PranitaPJun20 
Here you need to write

Line 39 to 42

What you write that is also option but we discuss Try to check with in if condition.

**Review by harshvegada — CHANGES_REQUESTED** [review]
Please do change as per comment

File: src/pranitaPJun20/MaxFreqWordString_Collections.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/MaxFreqWordString_Collections.java:28]: Ok Harsh.Will update my code

**Review by harshvegada — APPROVED** [review]
Looks good pranita
Keep it up 👍

**Pull request merged**